### DISCUSSION TITLE
A small demo of some Krylov.jl methods not discussed in tutorials

### DISCUSSION THREAD

**Discussion by mbyers31** [JuliaSmoothOptimizers/Krylov.jl#460] [Show and tell]
I created a short notebook demonstrating some of the methods available in Krylov.jl, along with a comparison of the methods at the end. You can find the notebook [on my github here](https://github.com/cu-numpde/2021-community-mbyers31/blob/main/krylov_demo.ipynb) .  If you're interested, you're free to use this code as-is or refactor it into a tutorial for your documentation. I used three methods: BiLQ, BiLQR, and QMR, none of which I could find a tutorial for in the [documentation](https://juliasmoothoptimizers.github.io/Krylov.jl/stable/examples/). 

Thanks for the great library!

**Comment by amontoison** [comment]
Thanks for your great tutorial @mbyers31 !
I will add it in the documentation this week :+1: 
You have a small typo with `r_y` for information, it's `r_y = c - A' * y`.
